src/main/java/command/Command.java

```diff
@@ -0,0 +1,61 @@
+package command;
+
+import exception.DukeException;
+import storage.Storage;
+import task.TaskList;
+import ui.Response;
+import ui.Ui;
+
+/**
+ * Represents an abstraction of sub-types.
+ * A command is a parent to the different kinds of commands
+ * a user can execute - todo/deadline/event/delete/exit/incorrect/list/mark/unmark.
+ */
+public abstract class Command {
+
+    private static boolean isExit = false;
+    protected Response response;
+    /**
+     * Default class constructor.
+     */
+    public Command() {
+    }
+
+    /**
```

Comment:
Shouldn't there be no line separating the class methods from other methods? 🤔  I noticed the same issue in several other places too.